scrapers/poland.py

```diff
@@ -3,28 +3,87 @@
 from utils.constants import OUTPUT_DIR
 from utils.utils import get_website_content, gmaps_url_to_lat_lon, write_to_json, normalize
 
+POLAND_EN_URL = 'https://www.gov.pl/web/udsc/ukraina-en'
 POLAND_PL_URL = 'https://www.gov.pl/web/udsc/ukraina2'
 
-
 def scrape_poland_pl():
-  """Runs the scraping logic."""
-  content = get_website_content(POLAND_PL_URL)
-  core = get_core(content)
-  reception_arr = get_reception_points(content)
-  path = os.path.join(OUTPUT_DIR, 'poland_pl.json')
-  write_to_json(path, core, reception_arr, POLAND_PL_URL)
+  """calls scrape_poland with the appropriate arguments for the pl website"""
+  scrape_poland(POLAND_PL_URL, 'pl')
 
+def scrape_poland_en():
+  """calls scrape_poland with the appropriate arguments for the en website"""
+  scrape_poland(POLAND_EN_URL, 'en')
 
-def get_core(content):
+def get_core(content, locale):
   """Gets the content from a bullet points list of general information for Ukrainian citizens."""
-  items = content.find('div', class_="editor-content").findAll('li')
+  items = content.find('div', class_="editor-content").findAll("span" if locale == "en" else "li")
   text_arr = []
   for item in items:
+    if item.find(text="RECEPTION POINT ADDRESS"):
+      break
     text_arr.append(normalize(item.get_text(strip=True, separator=' ')))
   return text_arr
 
+def scrape_poland(url, locale):
+  """Runs the scraping logic."""
+  content = get_website_content(url)
+  core = get_core(content, locale)
+  if locale == "pl":
+    reception_arr = get_reception_points_pl(content)
+  else:
+    reception_arr = get_reception_points_en(content)
+  path = os.path.join(OUTPUT_DIR, f'poland_{locale}.json')
+  write_to_json(path, core, reception_arr, url)
+
+def get_reception_points_en(soup):
+  """Gets the list of reception points."""
+  items = soup.find('div', class_="editor-content").find('div').findChildren(recursive=False)
+  reception_list_start = False
+  recep_arr = []
+  count = 0
+
+  for item in items:
+    # start scraping for reception points after the title
+    if item.find(text="RECEPTION POINT ADDRESS"):
+      reception_list_start = True
+      continue
+    # stop scraping after "what's next?"
+    elif item.find(text="What next?"):
+      break
+
+    if reception_list_start:
```

Comment:
since everything under here is executed only if this contidion is true, I would do early-return like construction, i.e.
```
if not reception_list_start:
    continue
```
to lower identation a bit and make it more readable

Reply:
also fixed in [a47d317](https://github.com/Ukraine-Relief-Efforts/scraper/pull/31/commits/a47d3173325abdb976551414363ac675f480b6f0)